library/src/main/java/com/datatorrent/lib/db/jdbc/JdbcPOJOInputOperator.java

```diff
@@ -97,6 +97,9 @@
 
   protected int pageNumber;
 
+  @AutoMetric
+  protected long tuplesRead;
```

Comment:
As suggested by Gaurav  keeping it protected so that the subclass can update the metrics if required.